labels/de-DE.json

```diff
@@ -594,5 +594,12 @@
   "width": "Breite",
   "wrongCountryForType": "Bitte wählen Sie ein Land für diesen Kontakt Typ unterstützt",
   "yourAccount": "Ihr Konto",
-  "yourOrderNumber": "Ihre Auftragsnummer ist <strong> {0} </ strong>."
+  "yourOrderNumber": "Ihre Auftragsnummer ist <strong> {0} </ strong>.",
```

Comment:
The localization file is `DE-de` which is for German. Your text is in French. Please update this to be German as it should be. Ignore that some other labels are incorrectly in French.

You should also add the labels in alphabetical order in the right place in the file. The label `quoteInReview` should be after `quoteHistory` for example. Please add the labels in the right place. Thanks.

Reply:
Done.